package.nls.json

```diff
@@ -13,5 +13,7 @@
 	"hexEditor.goToOffset": "Go To Offset",
 	"hexEditor.selectBetweenOffsets": "Select Between Offsets",
 	"hexEditor.switchEditMode": "Switch Edit Mode",
+	"hexEditor.copyOffsetAsDec": "Copy Offset as Dec",
```

Comment:
Can we use "Address" instead of offset, and spell out "Decimal"?

```suggestion
	"hexEditor.copyOffsetAsDec": "Copy Address as Decimal",
```

Reply:
I left it as Offset as other commands, such as goToOffset, use offset.

Reply:
Ah, true. Let's just stick with that terminonlogy then, though can we say "Decimal" instead of "dec"?

Reply:
Sorry, I will change it to decimal.